docs/contributor_guide/development_process.rst

```diff
@@ -234,6 +234,35 @@ To run the same tests locally, find the corresponding pipeline definition (a
 the command to execute the tests (usually starting with
 :code:`python -m pytest`) or it refers to a template file with the command.
 
+Building the website
+^^^^^^^^^^^^^^^^^^^^
+
+The website is built using `Sphinx <https://www.sphinx-doc.org/en/master/>`_
+and some of its extensions. Specifically, the website is available for all our
+releases to allow users to check the documentation of the version of the
+package that they are using.
+
+When making changes to the documentation at least run the following command
+to build the website using your changes:
+
+.. code-block::
+
+    python -m sphinx -v -b html -n -j auto docs docs/_build/html
```

Comment:
Could we have `make` aliases for these?

Reply:
Sure thing! I made `make doc` the single version doc build command, and `make doc-multiversion` the full one. I hope that is reasonable, otherwise I'm open to other suggestions, of course.